hackathon_site/hardware/models.py

```diff
@@ -0,0 +1,60 @@
+from django.db import models
+from event.models import TeamEvent
+
+# Create your models here.
+
+
+class Category(models.Model):
+    name = models.CharField(max_length=255, null=False)
+    max_per_category = models.IntegerField(null=False)
+
+    created_at = models.DateTimeField(auto_now_add=True, null=False)
+    updated_at = models.DateTimeField(auto_now=True, null=False)
+
+    def __str__(self):
+        return self.name
+
+
+class Hardware(models.Model):
+    name = models.CharField(max_length=255, null=False)
+    model_number = models.CharField(max_length=255, null=False)
+    manufacturer = models.CharField(max_length=255, null=False)
+    datasheet = models.TextField(null=False)
+    quantity_available = models.IntegerField(null=False)
+    notes = models.TextField(null=False)
+    max_items_per_team = models.IntegerField(null=False)
+    picture = models.TextField(null=False)
```

Comment:
This isn't an upload tho, it already exists on the server and we just want to fetch the location

Reply:
It will be an upload, via a form in the admin site most likely. Even if not, we can set the location stored in this field manually, and the field provides a number of convienence methods for working with the referenced file: https://docs.djangoproject.com/en/3.0/ref/models/fields/#filefield.